docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -354,12 +372,12 @@ NOTE: While the following examples use the _default_ configuration, you can use
 ==== Cipher suites
 
 Cipher suites are the list of ciphers that you can use during the TLS handshake.
-You can configure the ordered list of enabled cipher suites. 
+You can configure the ordered list of enabled cipher suites.
 If not configured, a reasonable default is selected from the built-in ciphers.
```

Comment:
What does "reasonable" mean here? Not very informative for the reader, IMHO. And if the specific definition of how the default is selected is not important, perhaps this sentence should not be here at all.